v21.2/primary-key.md

```diff
@@ -26,7 +26,7 @@ You can [change the primary key](#changing-primary-key-columns) of an existing t
  Parameter | Description
 -----------|-------------
  `table_name` | The name of the table you're creating.
- `column_name` | The name of the Primary Key column.
+ `column_name` | The name of the Primary Key column. For [multi-region tables](multiregion-overview.html#table-localities), you can use the `crdb_region` column within a composite primary key in the event the original primary key may contain non-unique entries across multiple, unique regions.
```

Comment:
Is there a reason for `multiregion-overview.html#table-localities` vs `multiregion-overview.html#table-locality`? LGTM otherwise!

Reply:
@gemma-shay The header is named `table-locality` in 21.1 and `table-localities` in 21.2 and 22.1. Not sure why, but I wanna avoid breaking existing links. TFTR!